test/types-multipart.spec.js

```diff
@@ -267,12 +285,18 @@ describe('types-multipart', () => {
                     'Content-Type: application/json',
                     '',
                     '{}',
+                    '------WebKitFormBoundaryTB2MiQ36fnSJlrhY',
+                    'Content-Disposition: form-data; name="cont2"; filename=""',
+                    'Content-Type: text/javascript',
+                    '',
+                    'console.log("hello world");',
```

Comment:
is this intentional?

Reply:
What do you mean exactly? This tests whether it's possible to use a non-json text field with empty filename. Or do you think this test isn't necessary?